tests/unit/mixins.js

```diff
@@ -64,6 +64,35 @@ function unmount (container) {
 }
 
 describe('PeriodicActions', function () {
+    it('has a custom validator for context.executeAction', function () {
+        var contextTypes = PeriodicActionsMixin.contextTypes;
+        var executeActionValidator = null;
+
+        expect(contextTypes).to.be.an('object');
+
+        executeActionValidator = contextTypes.executeAction;
+        expect(executeActionValidator).to.be.a('function');
+
+        // Test inputs
+        [
+            false,
+            true,
+            null,
+            undefined,
+            0,
+            10,
+            '',
+            'nope',
+            /test/,
+            {},
+            []
+        ].forEach(function (value) {
+            expect(executeActionValidator({executeAction: value}, 'executeAction')).to.be.an.instanceof(Error);
+        });
+
+        expect(executeActionValidator({executeAction: function () {}}, 'executeAction')).to.be.an('undefined');
```

Comment:
Will do, hadn't even seen that, I searched the chai docs and found that first:
http://chaijs.com/api/bdd/#a-section

Had I scrolled down I'd have seen what you're talking about:
http://chaijs.com/api/bdd/#undefined-section